### PR TITLE
     master merge to feature branch at e9c0454

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

## Changes

* `master` merge to feature branch at e9c04544f

### CODE REVIEW

File: packages/es-client/indexer.js [hunk 1]
```diff
@@ -74,7 +74,13 @@ async function genericRecordUpdate(esClient, id, doc, index, type, parent) {
 
   // adding or replacing record to ES
   const actualEsClient = esClient || (await Search.es());
-  const indexResponse = await actualEsClient.index(params);
+  let indexResponse;
+  try {
+    indexResponse = await actualEsClient.index(params);
+  } catch (error) {
+    logger.error(`Error thrown on index ${JSON.stringify(error)}`);
```
Comment (botanical) [inline packages/es-client/indexer.js:81]: Nice ⭐

File: bamboo/set-bamboo-env-variables.sh [hunk 2]
```diff
@@ -33,8 +35,8 @@ declare -a param_list=(
   "bamboo_SECRET_LAUNCHPAD_PASSPHRASE"
   "bamboo_SECRET_METRICS_ES_PASS"
   "bamboo_SECRET_NPM_TOKEN"
-  "bamboo_SECRET_ORCA_POSTGRES_USER_PASSWORD"
   "bamboo_SECRET_ORCA_DATABASE_APP_USER_PASSWORD"
+  "bamboo_SECRET_ORCA_POSTGRES_USER_PASSWORD"
   "bamboo_SECRET_PROVIDER_FTP_PORT"
   "bamboo_SECRET_PROVIDER_HTTP_PORT"
   "bamboo_SECRET_RDS_ADMIN_ACCESS_SECRET_ARN"
```
Comment (botanical) [inline bamboo/set-bamboo-env-variables.sh:42]: alphabetizing 🤩

Comment (Jkovarik) [reply bamboo/set-bamboo-env-variables.sh:42]: That's courtesy of @flamingbear  pointing it out :)

**Review by botanical — APPROVED** [review]
looks to good me, thanks for bringing this into the feature branch!

**Pull request merged**